getting-started/scaling-out.md

```diff
@@ -86,11 +86,15 @@ foreign data wrapper and any data node (server) objects.
 Deleting a data node is done by calling `delete_data_node`:
 
 ```sql
-SELECT delete_data_node('node1');
+SELECT delete_data_node('node1', cascade=>'true');
```

Comment:
should remove the quotes around true

Reply:
Sorry, blindly copied from #203. Fixed

Reply:
Actually it works correctly with quotes too, so it doesn't require to fix #203. But ofc it is cleaner without them.
Thanks for quick approval.

Reply:
yes it works with quotes as well but since they are not needed they should be removed from #203 too i think